### ISSUE TITLE
Inscribe Axis

### ISSUE THREAD

**Issue by DerTricker** [eseifert/gral#130]
Hello everyone,

first of all: Thanks for developing such a nice plotting library :-)

I just managed to plot my first function with gral. It worked pretty good so far. But now I'd like to inscribe  the axis (in my case the x-axis is the time axis and the y-axis is the energy-axis). I didn't find any clues in the documentation. Shouldn't there be a Axis.setName or anything like that?

Thanks for your help.

Greetings from Germany

**Comment by seifertm** [comment]
You are welcome. It's great to see people using GRAL!

TL;DR
``xyPlot.getAxisRenderer(XYPlot.AXIS_X).getLabel().setText("Text under X-Axis")``

GRAL follows Java's notion of renderers. The `Axis` class is a data class and only allows to set values like the minimum and maximum values. It has nothing to do with the rendering on screen.
For the rendering, a class of type `AxisRenderer` is responsible. Different implementations are available, since you could display the scale linearly (`LinearRenderer2D`) or logarithmically (`LograithmicRenderer2D`), for example. The `AxisRenderer` for a specific `Axis`  can be retrieved using `Plot.getAxisRenderer(String)`, where the `String` represents the name of the axis. The implementations of `Plot` already have built-in String constants for different axes, such as `XYPlot.AXIS_X`.
From an `AxisRenderer`, you can retrieve a `Label` using `getLabel()`, which can be rotated, aligned, and so on, and also contains the text that is displayed near the axis. Just set the label text using the method `Label.setText(String)`.

**Comment by DerTricker** [comment]
Thank you so much for the detailed explanation!
It worked perfectly :-)

**Comment by eseifert** [comment]
I added this to the wiki as a [how to add an axis title](https://github.com/eseifert/gral/wiki/howto-axis-title).
